tests/unit/s2n_3des_test.c

```diff
@@ -57,7 +57,7 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(s2n_hmac_init(&conn->active.server_record_mac, S2N_HMAC_SHA1, mac_key, sizeof(mac_key)));
     conn->actual_protocol_version = S2N_TLS11;
 
-    int max_aligned_fragment = S2N_DEFAULT_FRAGMENT_LENGTH - (S2N_DEFAULT_FRAGMENT_LENGTH % 8);
+    int max_aligned_fragment = S2N_SMALL_FRAGMENT_LENGTH - (S2N_SMALL_FRAGMENT_LENGTH % 8);
```

Comment:
nitpick: If `conn->max_fragment_size` is going to be our source of truth for a given connection, it might be better to use that value instead of assuming `S2N_SMALL_FRAGMENT_LENGTH` is the default.